proposals/kubeflow-steering-committee-charter.md

```diff
@@ -0,0 +1,148 @@
+1. Receive and handle reports about code of conduct violations and maintain confidentiality.
+1. Act as the final escalation point and arbitrator for any disputes, issues, clarifications, or escalations within the project scope.
+
+## Delegated authority
+
+KSC may choose to delegate its authority to other committees as-needed.
+
+The committee currently recognizes this delegated authority for [To be set up]:
+
+-  Technical guidance is delegated to the [Technical Oversight Committee](https://github.com/kubeflow/community/blob/master/proposals/TECH-OVERSIGHT-COMMITTEE.md).
+-  Trademark guidance is delegated to the [Conformance Committee](https://github.com/kubeflow/community/blob/master/proposals/CONFORMANCE-COMMITTEE.md).
+
+## Committee Meetings
+
+KSC currently meets at least bi-weekly, or as-needed. Meetings are open to the public and held online, unless they pertain to sensitive or previleged matters. Examples of such matters are:
+- Privacy related issues
+- Private emails to the committee
+- Code of conduct violations
+- Certain Escalations
+- Disputes between members
+- Security reports
+
+Meeting notes are available to members of the kubeflow-discuss mailing list, unless community member privacy requires otherwise. Public meetings will be recorded and the recordings made available publicly.
+
+Questions and proposals for changes to governance are posted as issues in the kubeflow/community repo, and the KSC invites your feedback there. See [Getting in touch](#getting-in-touch) for other options.
+
+## Committee members
+
+KSC is composed of 5 (five) members. They are elected according to the election policy [TODO: add link].  
+Seats on the Steering Committee are held by an individual, not by their employer.  
+
+The current membership of the committee is (listed alphabetically by first name):
+
+<table>
+  <thead>
+    <tr>
+      <th><br>
+Member</th>
+      <th><br>
+Organization</th>
+      <th><br>
+Github Profile</th>
+      <th><br>
+Term Start</th>
+      <th><br>
+Term End</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+  </tbody>
+</table>
+
+## Emeritus Committee Members
+
+[This section will be populated when there are retired committee members.]
+
+## Decision process
+
+The steering committee desires to always reach consensus.
+
+### Normal decision process
+
+Decisions requiring a vote include:
+- Issuing written policy
+- Amending existing written policy
+- Creating, removing, or modifying a working group
```

Comment:
What about accepting or removing Kubeflow components ?

Reply:
I can add that. It's probably impossible to have a complete list here, so we'll just need to update this list as needed.